protos/google/cloud/speech/v1p1beta1/cloud_speech.proto

```diff
@@ -295,19 +295,11 @@ message RecognitionConfig {
   // version of the specified model does not exist, then the speech is
   // recognized using the standard version of the specified model.
   //
-  // Enhanced speech models require that you enable audio logging for
-  // your request. To enable audio logging, set the `loggingConsentState` field
-  // to ENABLED in the [GoogleDataCollectionConfig][google.cloud.speech.v1p1beta1.GoogleDataCollectionConfig] section of your request.
-  // You must also opt-in to the audio logging alpha using the instructions in
-  // the [alpha documentation](/speech/data-sharing). If you set `use_enhanced`
-  // to true and you have not enabled audio logging, then you will receive
-  // an error.
+  // Enhanced speech models require that you opt-in to the audio logging using
+  // instructions in the [alpha documentation](/speech/data-sharing). If you set
+  // `use_enhanced` to true and you have not enabled audio logging, then you
+  // will receive an error.
   bool use_enhanced = 14;
-
-  // *Optional* Contains settings to opt-in to allow Google to
-  // collect and use data from this request to improve Google's products and
-  // services.
-  GoogleDataCollectionConfig google_data_collection_opt_in = 10;
```

Comment:
This is an incompatible change. We must be very careful not to allow incompatibilities even in beta endpoints, unless really needed. @jonparrott why did they do it?

Reply:
Our hands are pretty tied here. We can tell them to stop doing this, but ultimately we need to shit the updated library.